packages/legend-query/src/stores/QueryParametersState.ts

```diff
@@ -114,7 +115,7 @@ export class QueryParameterState {
       mockParameterValue: action,
     });
     this.queryParameterState = queryParameterState;
-    this.parameter = variableExpression;
+    this.parameter = observe_VariableExpression(variableExpression);
```

Comment:
Ideally the observable should already be done before calling the constructor. I think it might be better to do.
new QueryParameterState(queryParameterState, observe_VariableExpression(variableExpression)), instead of adding the logic in the constructor.